.testing/tools/parse_fms_clocks.py

```diff
@@ -60,18 +60,34 @@ def main():
         print(json.dumps(config))
 
 
-def parse_mom6_param(param_file):
+def parse_mom6_param(param_file, header=None):
     params = {}
     for line in param_file:
         param_stmt = line.split('!')[0].strip()
```

Comment:
`param_stmt` is a string used as a bool in the logic, how about `if line.find("!") == 0`

Reply:
Thanks for looking over this, @raphaeldussin.  My Python may be a bit old fashioned.  Is the implicit `bool(string)` operation frowned upon these days?  (I tend to use `if var` too often, though I had thought this one was safe.)

Reply:
FWIW, the PEP8 style guide suggests using `if not str:` to check if a string is empty:

https://peps.python.org/pep-0008/#programming-recommendations

> For sequences, (strings, lists, tuples), use the fact that empty sequences are false:

```
# Correct:
if not seq:
if seq:

# Wrong:
if len(seq):
if not len(seq):
```